mindjunkies/lecture/tests/test_signals.py

```
@@ -0,0 +1,182 @@
+        
+        # Now should be 100% (2/2) if recalculated
+        # Note: This test will fail unless you implement a post_delete signal
+        # self.assertEqual(self.enrollment.progression, 100)
+        
+    def test_progression_with_deleted_completion(self):
+        """Test that progression is recalculated when a completion is deleted."""
+        # This would require a post_delete signal handler for LectureCompletion
+        
+        # Create completions for two lectures
+        completion1 = LectureCompletion.objects.create(user=self.student, lecture=self.lecture1)
+        completion2 = LectureCompletion.objects.create(user=self.student, lecture=self.lecture2)
```

Ruff (0.8.2)

Local variable `completion2` is assigned to but never used

Remove assignment to unused variable `completion2`

(F841)